Lib/py_compile.py

```diff
@@ -137,6 +137,10 @@ def compile(file, cfile=None, dfile=None, doraise=False, optimize=-1):
     except FileExistsError:
         pass
     source_stats = loader.path_stats(file)
+    sde = os.environ.get('SOURCE_DATE_EPOCH')
+    if sde and source_stats['mtime'] > int(sde):
+        source_stats['mtime'] = int(sde)
+        os.utime(file, (source_stats['mtime'], source_stats['mtime']))
```

Comment:
Is actually updating the source code's timestamp part of the spec? I couldn't find any mention of doing that anywhere else. Is this what other languages and build tools are doing?

Reply:
OK, then I say remove this and update the test to only care about what the generated `.pyc` file has as a timestamp.